extensions/wikia/PhalanxII/classes/PhalanxStatsPager.class.php

```diff
@@ -45,21 +45,24 @@ function getPagingQueries() {
 	}
 
 	function formatRow( $row ) {
-		$type = implode( ", ", Phalanx::getTypeNames( ( isset( $row->ps_blocker_hit ) ) ? $row->ps_blocker_hit : $row->ps_blocker_type ) );
+		$type = implode( ", ", Phalanx::getTypeNames( $row->ps_blocker_hit ?? $row->ps_blocker_type ) );
 		$username = $row->ps_blocked_user;
-		$timestamp = $this->app->wg->Lang->timeanddate( $row->ps_timestamp );
-		$oWiki = WikiFactory::getWikiById( $row->ps_wiki_id );
-		$url = ( isset( $row->ps_referrer ) ) ? $row->ps_referrer : "";
+		$timestamp = $this->getLanguage()->timeanddate( $row->ps_timestamp );
+		$oWiki = WikiFactory::getWikiByID( $row->ps_wiki_id );
+		$url = $row->ps_referrer ?? "";
 		$url = ( empty( $url ) && isset( $oWiki ) ) ? $oWiki->city_url : $url;
 
+		// SUS-184: Render usernames containing spaces correctly
+		$encUserName = str_replace(' ', '_', $username );
+
 		$specialContributionsURL = GlobalTitle::newFromText( 'Contributions', NS_SPECIAL, $row->ps_wiki_id )->getFullURL();
 
 		if ( !empty( $specialContributionsURL ) ) {
-			$username = '[' . $specialContributionsURL . '/' . $username . ' ' . $username . ']';
+			$username = '[' . $specialContributionsURL . '/' . $encUserName . ' ' . $username . ']';
 		}
 
 		$html  = Html::openElement( 'li' );
-		$html .= wfMessage( 'phalanx-stats-row', $type, $username, $url, $timestamp )->parse();
+		$html .= $this->msg( 'phalanx-stats-row', $type, $username, $url, $timestamp )->parse();
 		$html .= Html::closeElement( 'li' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$html". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)